uktrade/data-hub-frontend
# Add the FieldTypeahead component

## Description of change
This just moves the `<FieldTypeahead>` component from Data Hub Components to this repository.

## Test instructions
1. Run storybook and you should now see this component added into the left menu as it was in Data Hub Components.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
